samples/ngIdentity/README.md

```diff
@@ -0,0 +1,114 @@
+   ```csharp
+   builder.Services.AddAuthentication(IdentityConstants.ApplicationScheme)
+    .AddIdentityCookies()
+    .ApplicationCookie!.Configure(opt => opt.Events = new CookieAuthenticationEvents()
+    {
+        OnRedirectToLogin = ctx =>
+        {
+            ctx.Response.StatusCode = 401;
+            return Task.CompletedTask;
+        }
+    }); 
+    builder.Services.AddAuthorizationBuilder();
+   ```
+1. Configured the EF Core `DbContext` to use the in-memory identity database, and turned on the identity APIs.
+   ```csharp
+   builder.Services.AddDbContext<AppDbContext>(
+      options => options.UseInMemoryDatabase("AppDb"));
+
+   builder.Services.AddIdentityCore<MyUser>()
+     .AddEntityFrameworkStores<AppDbContext>()
+     .AddApiEndpoints();
+   ```
+1. Mapped the identity API endpoints: `app.MapIdentityApi<MyUser>();`
+1. Added a logout method. This isn't included in the endpoints because it may change based on your configuration (i.e. cookies, tokens, both). More on this method below.
+1. Locked down the weather forecast API to only respond to authenticated users by adding the extension: `.RequireAuthorization();`
+
+The `logout` endpoint is defined like this:
+
+```csharp
+app.MapPost("/logout", async (
+    SignInManager<MyUser> signInManager,
+    [FromBody]object empty) =>
+{
+    if (empty is not null)
+    {
+        await signInManager.SignOutAsync();
+        return Results.Ok();
+    }
+    return Results.NotFound();
+}).RequireAuthorization();
+```
+
+Minimal APIs allow you to bind to forms. When you bind to forms, you risk exposing your site to cross-site request forgery (CSRF) attacks. To prevent this, you use the [anti-forgery services](https://learn.microsoft.com/aspnet/core/security/anti-request-forgery?view=aspnetcore-8.0). This logout doesn't bind to a form, but it also takes no parameters. Without parameters, an empty forms post from a malicious site could invoke the API. To avoid this, the client will post with an empty body: `{}`. The server checks for the parameter. If the empty body is included, the `empty` parameter is not null (it is deserialized as a `JObject`) and the method will succeed.
+
+A forms post cannot produce the empty JSON body, so will either render a null parameter or throw a `415` (media type not supported). There is an option on the home page of the client app to try to log out using the cross-site forms post. 
+
+### Client
+
+The components to manage authentication are in the `identity` folder.
+
+* **dto** defines the shape of parameters for the identity APIs.
+* **guard** can be used with the router to prevent navigation to a route if the user is not authenticated. It will automatically redirect to the login page. It is used to protect the weather forecast route (although the server will also prevent it from being accessed by an unauthenticated user).
```

Comment:
```suggestion
* **guard** can be used with the router to prevent navigation to a route if the user is not authenticated. It automatically redirects to the login page. It's used to protect the weather forecast route, although the server also prevents it from being accessed by an unauthenticated user.
```